src/databricks/labs/ucx/framework/dashboards.py

```diff
@@ -149,9 +175,9 @@ def _installed_query_state(self):
             self._state = json.load(self._ws.workspace.download(self._query_state))
             to_remove = []
             for k, v in self._state.items():
+                _, name = k.split(":")
```

Comment:
This may contain 3 elements now, account for it

Reply:
Wait what? Since when? Where?